views/index.html

```diff
@@ -15,8 +15,8 @@
 
     <script type="text/javascript">
       // Client ID and API key from the Developer Console
-      var CLIENT_ID = '<YOUR_CLIENT_ID>';
-      var API_KEY = '<YOUR_API_KEY>';
+      var CLIENT_ID = '396870052791-b9ul5pd3p37la7j00u0nlabnor76rapj.apps.googleusercontent.com';
+      var API_KEY = 'AIzaSyCnJd0tY_HSPnrKTYn47bmkg6ZEATW_uBI';
 
```

Comment:
Never push `API Keys` and tokens. 
Please remove this. 
```js
var CLIENT_ID = 'XXXXXXXXXXXXX',
var API_KEY = 'XXXXXXXXXXXXX';
```
@PriyankVIT

Reply:
@PriyankVIT remove the API Keys, I will merge once this is done